src/main/java/sqlline/SqlLine.java

```diff
@@ -1603,11 +1601,6 @@ int print(ResultSet rs, DispatchCallback callback) throws SQLException {
       }
     }
 
-    if (f == null) {
```

Comment:
This check becomes useless as after this commit the check for valid output format is going to be done before property setting